lib/plugin.js

```diff
@@ -7,16 +7,23 @@ window.io = require('socket.io-client')
 <% } %>
 
 function getHeaders ({ app }) {
-  const headers = {}
+  const headers = {
+    referer: location.origin
```

Comment:
@JoaoPedroAS51 What's the purpose of setting the referer header here? Chrome is logging the error `Refused to set unsafe header "referer"` every time I connect to a channel via Pusher. Is this required for something?

Reply:
If I'm not wrong, it is required for Laravel Sanctum.

Reply:
I'm using Passport and it's definitely not required there. Would it make more sense to move it down to line 19 in the Sanctum conditional? The package does work properly with it here (at least in my usage), but moving it would remove the errors in the Chrome console.

Reply:
Yeah, you're right! If it's only required for Sanctum, it should be moved to Sanctum conditional. I'll do that tomorrow :)

Reply:
Thanks for your help reviewing and testing the code!

Reply:
Fixed in 66e0111